packages/terra-button/src/Button.module.scss

```
@@ -32,6 +32,7 @@
     font-family: inherit;
     font-size: var(--terra-button-font-size, 1rem);
     line-height: var(--terra-button-line-height, 1.429);
+    margin: 0; /* Reset the margin in Safari. */
```

nit: want to add more context to the comment?
```suggestion
    margin: 0; /* Reset margin for Safari. Safari defaults to 2px */
```

---

Done - https://github.com/cerner/terra-core/pull/2316/commits/8b4ce473766ef3d3bcc2e3a7f8fc71df2ed25756